client/src/App.js

```diff
@@ -1,7 +1,76 @@
 import React from 'react';
+import styled from '@emotion/styled';
+import Particles from 'react-particles-js';
+import { useState, useEffect } from 'react';
+import colors from './utils/colorsUtils';
+
+function getWindowDimensions() {
+  const { innerWidth: width, innerHeight: height } = window;
+  return {
+    width,
+    height,
+  };
+}
+
+function useWindowDimensions() {
```

Comment:
good job with the `useWindowDimensions` hook.
I would outsource it with `getWindowDimensions`.

Reply:
I will outsource it in the near future. This is just a first test, to understand how things work.